### PR TITLE
[ docs ] Add emacs mode keybindings to user manual

### PR DESCRIPTION
**Pull request by adamse**
Adds keybinds to manual, pretty much copied from http://wiki.portal.chalmers.se/agda/pmwiki.php?n=Docs.EmacsModeKeyCombinations with the addition of computing helper function types.

Also adds an empty section on Auto (under Tools).

### CODE REVIEW

File: doc/user-manual/tools/auto.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.. _auto:
+
+****
+Auto
```
Comment (phile314) [inline doc/user-manual/tools/auto.rst]: I think the title "Auto" is not very descriptive. Maybe something along the lines of "Automatic Proof Search (Auto)" is better? On the other hand, auto does not only work for proofs but also for arbitrary program fragments in general...

File: doc/user-manual/tools/index.rst [hunk 2]
```diff
@@ -8,9 +8,9 @@ Tools
 .. toctree::
    :maxdepth: 2
 
+   auto
    emacs-mode
```
Comment (phile314) [inline doc/user-manual/tools/index.rst:11]: I would prefer having the auto section after emacs-mode.

**Comment by phile314** [comment]
Looks good in general. I made two minor comments inline.

**Comment by adamse** [comment]
Your comments made sense, I made the changes.

**Comment by phile314** [comment]
Thank you, is merged.

**Pull request merged**